docs/api/graphql_customization.md

```diff
@@ -10,18 +10,22 @@ You can customize the GraphQL schema that is generated from your repository.
 
 You can use it if your application requires custom GraphQL resources, for instance for Doctrine entities.
 
-To do so, create an `app/config/graphql/Query.types.yaml` file. It will be used as the GraphQL query root.
+To do so, create a `config/graphql/types/Query.types.yaml` file. It will be used as the GraphQL query root.
 
 In that file, add new fields that use any custom type or custom logic you require, based
 on [overblog/GraphQLBundle](https://github.com/overblog/GraphQLBundle).
 
+The custom schema should be created only after generating other schemas to avoid problems, especially if the custom schema depends on other schema elements. For example ...
```

Comment:
```suggestion
The custom schema should be created only after generating other schemas to avoid problems, especially if the custom schema depends on other schema elements. For example:
```